sycl/include/sycl/ext/intel/fpga_device_selector.hpp

```diff
@@ -22,6 +23,28 @@ class platform;
 namespace ext {
 namespace intel {
 
+namespace detail {
+// Scores a device by platform name.
+inline int selectDeviceByPlatform(std::string_view required_platform_name,
+                                  const device &device) {
+  const platform &pf = device.get_platform();
+  std::string platform_name = pf.get_info<sycl::info::platform::name>();
+  if (platform_name == required_platform_name) {
```

Comment:
nit: I'd probably just eliminate the temp variable.

Reply:
I have removed both.